exo-setup/src/docker-setup.ls

```diff
@@ -33,7 +33,8 @@ class DockerSetup
       environment: @_get-docker-env-vars!
       depends_on: @_get-service-dependencies!
     for dependency in @service-config.dependencies or []
-      docker-config[dependency.name + dependency.version] = @_get-service-dependency-docker-config dependency.name, dependency.version, dependency.config
+      if dependency.config
```

Comment:
Let's add a comment: 
```
# only add dependency to service if 'config' field exists,
# otherwise the dependency has already been listed as an application dependency
```